meetup/models.py

```diff
@@ -1,3 +1,62 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Venue(models.Model):
+    """This model is for the venue where be held"""
+    name = models.CharField(max_length=50)
+    location = models.CharField(max_length=255, null=True)
+    description = models.TextField(null=True)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+
+
+class ProgramCategory(models.Model):
+    """This model is for the category of program"""
+    name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=100, unique=True)
+
+
+class Speaker(models.Model):
+    """This model is for that user related by speaker at program"""
+    user = models.ForeignKey(User)
+
+
+class Program(models.Model):
+    """This model is for that lecture or event by speaker(s)"""
+    # There should be difficulty and language information in Program?
+    title = models.CharField(max_length=255)
+    brief = models.TextField(null=True, verbose_name='simple information')
+    description = models.TextField(null=True)
+    speakers = models.ManyToManyField(Speaker)
+    category = models.ForeignKey(ProgramCategory, verbose_name='The category which this program is belonged')
+    slide_url = models.CharField(max_length=255, null=True, verbose_name='A slide url')
+    pdf_url = models.CharField(max_length=255, null=True, verbose_name='A pdf url')
+    video_url = models.CharField(max_length=255, null=True, verbose_name='A video url')
+    is_recordable = models.BooleanField(default=True, verbose_name='recordable condition')
+
+    # If kind of session or lecture True, or not False (include breaktime)
+    is_main_event = models.BooleanField(default=True)
+
+
+class Profile(models.Model):
+    """This model is for which profile for user"""
+    user = models.ForeignKey(User)
+    name = models.CharField(max_length=50)
+    slug = models.SlugField(max_length=50, unique=True, verbose_name='nickname')
```

Comment:
Common use case of slug is make url for specific object. That is reason why slugfield cannot use space. But 'nickname' is nickname. Nickname need space, Korean character and special character also.
So make new field for nickname is more make sense to me.

Reply:
Then, can I use that nickname as the slug like that `Noh Seho` to `Noh-Seho`?
or the `slug` variable should be changed the name to `username`? The `username` what I think means that the `username` is not allowed space.

Reply:
If you have plan to provide url for each profiles then keep this `slug` field and just change verbose_name 😄 .
and also you can use nickname field as slug. But that is not a good idea in performance reason.

Reply:
ok, thank you